internal/chain/engine.go

```diff
@@ -480,42 +480,23 @@ func applySlashes(
 	// Do the slashing by groups in the sorted order
 	for _, key := range sortedKeys {
```

Comment:
Following the rating changes, we are not completely sure whether or not the records should remain grouped.
Since the new implementation doesn't take into consideration the voting power of the BLS keys that made the same double sign, from our understanding sorting can be applied directly to each record, and `apply()` can handle each slashing record independently. Any ideas on that @rlan35?